aries-site/src/components/content/index.js

```diff
@@ -1,4 +1,8 @@
 export * from './BulletedList';
+// was getting a linting error in controls.js that it couldn't
+// find figma anchor unless I exported as such.
+export { FigmaAnchor } from './FigmaAnchor';
 export * from './ComingSoon';
 export * from './SubsectionText';
 export * from './SubmitFeedback';
+export * from './StorybookAnchor';
```

Comment:
I understand what you're saying. I guess my thought was to avoid having to write the labels of "Use with Figma" or "See in Storybook" so many times. I am open to consolidating them. Do you have an idea for a clear name for that component?